lib/matplotlib/sankey.py

```diff
@@ -83,10 +83,11 @@ def __init__(self, ax=None, scale=1.0, unit='', format='%G', gap=0.25,
         unit : str
             The physical unit associated with the flow quantities.  If *unit*
             is None, then none of the quantities are labeled.
-        format : str
-            A Python number formatting string to be used in labeling the flow
-            as a quantity (i.e., a number times a unit, where the unit is
-            given).
+        format : str or callable
+            A Python number formatting string or callable used to label the
+            flows with their quantities. If a format string is given, the
+            label will be ``format % quantity``. If a callable is given, it
+            will be called with ``quantity`` as an argument.
```

Comment:
The original docstring contained this block
```
(i.e., a number times a unit, where the unit is given)
```
Is it right to think that the information of the original behaviour should be retained?
```
A Python number formatting string or callable used to label the flows with their quantities. (i.e., a number times a unit, where the unit is given)
If a format string is given, the label will be ``format % quantity``. If a callable is given, it will be called with ``quantity`` as an argument.
```
(Something like this?)

Reply:
> The original docstring contained this block

> 
> Is it right to think that the information of the original behaviour should be retained?
> 

> (Something like this?)

Ok, sure I updated the PR with this change. 

This seems fine to me, but also a little marginal, as other format related parameters don't seem to mention this in their docstrings, see:

https://github.com/matplotlib/matplotlib/blob/6305e8da0e57c99e3e860b3b030c3fa742947be9/lib/matplotlib/axes/_axes.py#L2942-L2946

https://github.com/matplotlib/matplotlib/blob/99e6240d95e86b4fd9edbcbefa8a5a135e707f27/lib/matplotlib/contour.py#L93-L101

Reply:
I read the comment about units to be sankey specific.